### ISSUE TITLE
Encoding is too slow. What can i do ? for it

### ISSUE THREAD

**Issue by cengizakcan** [ageitgey/face_recognition#951]
Hi,

what can i do for encoding ? 

because very slowly. and how can i use all cpu bcs it use only one core cpu.

**Comment by dktechindia** [comment]
Did you found any solution for this?

**Comment by irfans4536** [comment]
please help mei if anyone found any solution

**Comment by cengizakcan** [comment]
> Did you found any solution for this?

Yes.  i did it. but this is private. sorry. just think how can i compress.

**Comment by saya0769** [comment]
我也想知道.

**Comment by cengizakcan** [comment]
> 我也想知道.

對不起兄弟 這是私人作品，如果您想獲得支持，您必須支付薪水。
